### PR TITLE
update zh_TW translate

### PR DESCRIPTION
**Pull request by ibmibmibm**

### CODE REVIEW

**Comment by bonii-xx** [comment]
What's the difference between the regular and the bold punctuation marks?

**Comment by ibmibmibm** [comment]
The original version use a half-width marks and a space, while used a lot in China.
The version I submitted is using full-width marks, it's more common with Traditional Chinese within Taiwan and Hongkong.
Reference: https://en.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms

**Comment by asl97** [comment]
@bonii-xx It's not bold punctuation marks, it's [fullwidth punctuation marks](https://en.wikipedia.org/wiki/Chinese_punctuation).
It fit better into the style of the language.

@ibmibmibm In Chinese, [the ellipsis is six dots (in two groups of three dots)](https://en.wikipedia.org/wiki/Ellipsis#In_Chinese)

File: resources/assets/tconstruct/lang/zh_TW.lang [hunk 1]
```diff
@@ -429,24 +430,24 @@ item.tconstruct.fancy_frame.clear.name=通透物品展示框
...
 # Mom's Spaghetti #
 ###################
 
 item.tconstruct.moms_spaghetti.name=媽媽的義大利麵
-item.tconstruct.moms_spaghetti.desc=好吃!
-item.tconstruct.moms_spaghetti.tooltip1=§o我猜味道大概都對了... 但是還是有點不一樣。\n他們肯定需要一些醬汁，但它必須是紅色的。
-item.tconstruct.moms_spaghetti.tooltip2=§o少了一些東西... 可以加一些肉。 媽媽總是使用所有種類的肉。\n我應該取得至少三種以上不同的肉。
+item.tconstruct.moms_spaghetti.desc=好吃！
+item.tconstruct.moms_spaghetti.tooltip1=§o我猜味道大概都對了…但是還是有點不一樣。\n他們肯定需要一些醬汁，但它必須是紅色的。
+item.tconstruct.moms_spaghetti.tooltip2=§o少了一些東西…可以加一些肉。媽媽總是使用所有種類的肉。\n我應該取得至少三種以上不同的肉。
```
Comment (asl97) [inline resources/assets/tconstruct/lang/zh_TW.lang]: ellipsis

File: resources/assets/tconstruct/lang/zh_TW.lang [hunk 2]
```diff
@@ -614,134 +615,134 @@ stat.projectile.ammo.name=發數
 
 # traits are also modifiers, they use the same prefix so they don't have to be localized twice
 modifier.alien.name=外星人
-modifier.alien.desc=§o從性狀感覺起來...§r\n...它們好像正在改變! 也許時間會證明一切?
+modifier.alien.desc=§o從性狀感覺起來…§r\n…它們好像正在改變！也許時間會證明一切？
```
Comment (asl97) [inline resources/assets/tconstruct/lang/zh_TW.lang]: ellipsis

File: resources/assets/tconstruct/lang/zh_TW.lang [hunk 3]
```diff
@@ -614,134 +615,134 @@ stat.projectile.ammo.name=發數
...
+modifier.crude.desc=§o給膽小鬼的裝備！§r\n對沒有裝備的目標造成傷害。
 modifier.crude.extra=對付無裝備 : +%s
 modifier.crumbling.name=崩塌
 modifier.crumbling.desc=§o轟隆隆§r\n這個工具破壞軟方塊比沒拿工具還快。
 modifier.dense.name=稠密
-modifier.dense.desc=§o比較...硬。§r\n你的工具在耐久度很少的時候撐得比較久。
+modifier.dense.desc=§o比較…硬。§r\n你的工具在耐久度很少的時候撐得比較久。
 modifier.duritos.name=持久
-modifier.duritos.desc=§o用起來不像它聽起來那麼好。§r\n你的工具可以撐得比較久...大部分時候啦...
+modifier.duritos.desc=§o用起來不像它聽起來那麼好。§r\n你的工具可以撐得比較久…大部分時候啦…
```
Comment (asl97) [inline resources/assets/tconstruct/lang/zh_TW.lang]: ellipsis

File: resources/assets/tconstruct/lang/zh_TW.lang [hunk 4]
```diff
@@ -614,134 +615,134 @@ stat.projectile.ammo.name=發數
...
 modifier.superheat.name=過熱
-modifier.superheat.desc=§o增加熱源!§r\n在敵人著火時可以對敵人造成更多傷害。
+modifier.superheat.desc=§o增加熱源！§r\n在敵人著火時可以對敵人造成更多傷害。
 modifier.superheat.extra=對付燃燒者 : +%s
 modifier.unnatural.name=不自然
-modifier.unnatural.desc=§o令人憎恨!§r\n當你的工具挖礦等級高於你所挖的方塊時，這個工具挖掘得更快。
+modifier.unnatural.desc=§o令人憎恨！§r\n當你的工具挖礦等級高於你所挖的方塊時，這個工具挖掘得更快。
 modifier.tasty.name=好吃
-modifier.tasty.desc=§o(嚼嚼嚼...)§r\n好香啊... 你應該吃你的工具而不是挨餓。
+modifier.tasty.desc=§o(嚼嚼嚼…)§r\n好香啊… 你應該吃你的工具而不是挨餓。
```
Comment (asl97) [inline resources/assets/tconstruct/lang/zh_TW.lang]: ellipsis

File: resources/assets/tconstruct/lang/zh_TW.lang [hunk 5]
```diff
@@ -614,134 +615,134 @@ stat.projectile.ammo.name=發數
...
-modifier.emerald.desc=§o漂亮!§r\n基於工具基礎的性能增加耐久度!
+modifier.emerald.desc=§o漂亮！§r\n基於工具基礎的性能增加耐久度！
 modifier.fiery.name=火熱
-modifier.fiery.desc=§o現在好熱!§r\n造成火焰傷害，讓你的敵人著火。
+modifier.fiery.desc=§o現在好熱！§r\n造成火焰傷害，讓你的敵人著火。
 modifier.fiery.extra=火焰傷害 : +%s
 modifier.fiery.extra2=燃燒持續時間 : %ss
 modifier.fins.name=魚鰭
-modifier.fins.desc=§o有時候有點腥味...§r\n在投射物黏上魚鰭讓它就像水下生物一樣移動。
+modifier.fins.desc=§o有時候有點腥味…§r\n在投射物黏上魚鰭讓它就像水下生物一樣移動。
```
Comment (asl97) [inline resources/assets/tconstruct/lang/zh_TW.lang]: ellipsis

**Comment by bonii-xx** [comment]
Curious, the things you learn. Makes sense for that language, thanks for the info.

File: resources/assets/tconstruct/lang/zh_TW.lang [hunk 6]
```diff
@@ -427,33 +428,33 @@ item.tconstruct.fancy_frame.gold.name=黃金物品展示框
...
-modifier.spaghetti_meat.desc=§o所有的肉!§r\n增加回復的飢餓值
+modifier.spaghetti_meat.desc=§o所有的肉！§r\n增加回復的飢餓值
 
 item.tconstruct.spaghetti.hard.name=超硬的小麥棒
 item.tconstruct.spaghetti.hard.tooltip=就如它的名稱一樣不可食用。\n它必須被軟化，就算你的手使盡全力也不能軟化它。\n他們應該泡在水裡一段時間。
 item.tconstruct.spaghetti.soggy.name=濕麵條
 item.tconstruct.spaghetti.soggy.tooltip=現在，它已經濕透了，就算是你軟趴趴的手臂也能夠折彎它。\n讓我們把它掛起來晾乾。
 item.tconstruct.spaghetti.cold.name=乾冷麵條
-item.tconstruct.spaghetti.cold.tooltip=嗯，好多了。但是它吃起來還是很噁心。\n加熱它應該可以使它變得更好入口。\n甚至它可能好到可以讓你重新站起來。
+item.tconstruct.spaghetti.cold.tooltip=嗯，好多了。但是它吃起來還是很噁心。\n加熱它應該可以使它變得更好入口。\n甚至它可能好到讓你雙腿一軟。
```
Comment (3TUSK) [inline resources/assets/tconstruct/lang/zh_TW.lang]: 打扰一下：

> 甚至它可能好到讓你雙腿一軟

原文中 "... get you off your weak knees" 的 "get off" 似是“避免；免除”的意思，所以我认为原本的翻译可能更合适。
如有疏漏望指明。

**Pull request merged**